### PR TITLE
monorepo: update nyc and verdaccio

### LINKED ISSUE #832: Upgrade nyc to v15

### PR DESCRIPTION
**Pull request by emersonmacro**
First round for #1537, closes #832

### CODE REVIEW

**Review by ryanio — DISMISSED** [review]
lgtm, thanks!

File: package.json
Comment (holgerd77) [inline package.json:21]: Asking independently from this PR: is it necessary to have these fixed dependencies here? 🤔 (so also on `npm-auth-to-token`). In doubt we will always get in new vulnerabilties in like this.

If no one knows anything about it we should rather just test to add a `^`, can't really lead to anything grave here.

Comment (holgerd77) [reply package.json]: Update: I've just directly added here and will see how it goes.

**Review by holgerd77 — APPROVED** [review]
Re-approving after having dismissed the last approval by doing an additional commit.

Will merge.

**Pull request merged**
